### DISCUSSION TITLE
I am puzzled by slow-loading

### DISCUSSION THREAD

**Discussion by davidpassmore** [quarto-dev/quarto-cli#11009] [Q&A]
### Description

I am experiencing slow-loading of the out-of-the-box Quarto blog locally, through GitHub, and on Quarto Pub. 15-20 secs to load, even locally.
GitHub code is at (same as local and Quarto pub code): https://github.com/davidpassmore/shorttest

Here is the yaml file
````qmd
project:
  type: website
  output-dir: "docs"

website:
  title: "shortertest"
  navbar:
    right:
      - about.qmd
      - icon: github
        href: https://github.com/
      - icon: twitter
        href: https://twitter.com
format:
  html:
    theme: cosmo
    css: styles.css

editor: visual
````

Here is index.qmd
````qmd
---
title: "shortertest"
listing:
  contents: posts
  sort: "date desc"
  type: default
  categories: true
  sort-ui: false
  filter-ui: false
page-layout: full
title-block-banner: true
---

````
So, the loading wait-time is long everywhere.

I made a separate instance of a quarto blog. Same loading time.

I have another, small but more complex that loads quickly at https://davidpassmore.quarto.pub/simons/

Any suggestions to improve this glacial loading I am experiencing? I appreciate any advice you have.

Thank you,
David

**Comment by mcanouil** [comment]
May I suggest you upgrade to the latest stable version?
You are running 1.4.549 apparently which is not even the latest 1.4.

Also, please do share reproducible code.
The code you shared is clearly incomplete looking at the produced files in your repository.
For example:

```sh
quarto create project blog my-blog
```

**Reply by davidpassmore** [reply]
Thank you, mcanouil.

_Created a new project file using:_ quarto create project blog onetwonow
Still experiencing long wait to view index.html after I render.

I am attaching a .zip  of the project: onetwonow.zip
Perhaps you see something I am goofing up?

_Here is my R version:_
> R.version
               _                           
platform       x86_64-apple-darwin20       
arch           x86_64                      
os             darwin20                    
system         x86_64, darwin20            
status                                     
major          4                           
minor          4.1                         
year           2024                        
month          06                          
day            14                          
svn rev        86737                       
language       R                           
version.string R version 4.4.1 (2024-06-14)
nickname       Race for Your Life  

_Running Quarto 1.4.4 now:_
R Interface to 'Quarto' Markdown Publishing System [R logo]
[Up] [Top]
Documentation for package ‘quarto’ version 1.4.4
[onetwonow.zip](https://github.com/user-attachments/files/17295512/onetwonow.zip)

**Reply by mcanouil** [reply]
> Running Quarto 1.4.4

What is that? That's unlikely Quarto CLI.
https://quarto.org/docs/get-started/

To be clear Quarto is a command line interface and it exists independently of R/Python/Julia.
The R package is not doing anything except calling Quarto CLI.
Updating the R package has no effect on Quarto CLI.

**Reply by davidpassmore** [reply]
Thank you for the suggestion and advice about Quarto CLI.

I updated with the Quarto CLI at https://quarto.org/docs/get-started/. Updating helped. I am not experiencing the slow loading I was seeing.

For the future, must I examine the Quarto page to identify new CLI when available? Or, is some other notification possible? 

Appreciate your willingness to help me learn this.

David

**Reply by mcanouil** [reply]
Unfortunately, there are nothing in Quarto CLI letting you know there is an update.
You can go to Quarto website from time to time (twice a year would be a good basis).
Alternatively, you can subscribe on GitHub to releases, although the default is for all releases (not sure you can configure only for stable releases).

**Reply by cscheid** [reply]
> Unfortunately, there are nothing in Quarto CLI letting you know there is an update.

Let me explain this a bit. We do it by design. Quarto is used in a number of sensitive contexts, specifically in computing environments where internet access is disallowed. We have promised users that Quarto will never "phone home", in any meaningful sense, as a simple "bright red line" that covers the privacy requirements of such users. Yes, it would be possible to make Quarto "phone home" without providing any revealing information. But it would mean that such privacy-constrained users would need to audit their network logs to be satisfied that no information is leaking.

Instead, we make a deliberate choice to simply not have telemetry of any kind.

**Reply by davidpassmore** [reply]
TY, understand. Still learning.

I set up notifications about releases on the GitHub site.

I was using distill previously. Quarto is such great extension of everything I could do there.

Appreciate the help/instruction provided.

David
